src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1115,16 +1094,23 @@ k
             }
```

Comment:
`prop` _is_ defined, and it is being used. It's being defined at `App.js:10`, which make sense as to why it would be globally accessible. Left a todo there. 

done